akka-actor/src/main/mima-filters/2.5.22.backwards.excludes

```diff
@@ -0,0 +1,7 @@
+# These were added for bincompat with 2.4 but no longer part of the public API,
```

Comment:
file 2.5.23 or 2.5.x?

Reply:
Wondering the same.

Reply:
Are we back porting this? I assume so

Reply:
Yes, I think it should be whatever version we merge https://github.com/akka/akka/pull/26992 into